src/views/Workspace/Cards/CollectOpinionsCard.jsx

```diff
@@ -1,8 +1,39 @@
 import React from "react";
+import { withRouter } from "react-router-dom";
+import Typography from "@material-ui/core/Typography/Typography";
+
 import WorkspaceCard from "../../../components/CustomCard/WorkspaceCard";
+import { isUserVolunteerOfPlayground } from "../../../components/Playground/PlaygroundReducer";
+import { getWorkspaceStartLink } from "../../../misc/WorkspaceHelpers";
 
 // step: "Meningen Inventariseren"
 class CollectOpinionsCard extends React.Component {
+    componentDidMount() {
+        this.setCta();
+    }
+
+    componentDidUpdate(prevProps) {
+        const { user } = this.props;
+
+        if (!prevProps.user && user) {
+            this.setCta();
+        }
+    }
+
+    componentWillUnmount() {
+        this.props.unsetCta();
+    }
+
+    setCta() {
+        const { setCta, playground, user, history } = this.props;
+
+        setCta({
+            ctaAction: () => history.push(`${getWorkspaceStartLink(playground)}/team`),
+            ctaText: 'Chat with teammembers',
```

Comment:
'Chat with teammembers' translates into 'Chat met teamleden'

Reply:
clicking this button should also open up community category;

Reply:
Good catch. Thanks! Fixed.